src/components/GridListToggle.svelte

```diff
@@ -1,17 +1,20 @@
 <script>
 import Icon from 'fa-svelte'
-import { createEventDispatcher } from 'svelte'
 import { faThList, faTh } from '@fortawesome/free-solid-svg-icons'
+import { createEventDispatcher } from 'svelte'
 
-export let buttonCssClass = ''
-export let showAsList
+export let choice = 'grid'
 
 const dispatch = createEventDispatcher()
+const classes = `btn btn-sm ${$$props.class}`
```

Comment:
If the view using this was defining the `class` dynamically, would this need to be a `let`? We're not doing so at the moment. However, I wondered if using a const for a value dependent upon an input parameter was a little too strict.

Reply:
`$$props.class` is not reactive...should we talk through this a little then?